requery/src/main/java/io/requery/sql/RowCountException.java

```diff
@@ -24,11 +24,13 @@
  */
 public class RowCountException extends PersistenceException {
 
+    private final String entityName;
     private final long expected;
     private final long actual;
 
-    RowCountException(long expected, long actual) {
-        super("Expected " + expected + " row affected actual " + actual);
+    RowCountException(String entityName, long expected, long actual) {
```

Comment:
Looking at the usage it would be simpler if this argument was a `Class` instead of a `String`. That would simplify the usage in all the calling sites

Reply:
Thanks for the point, updated PR.